index.html

```diff
@@ -0,0 +1,234 @@
+      </nav>
+      <div class="menu-overlay hide-menu">
+        <ul class="mobile-menu-toggle menu-close">
+          <li><i class="fas fa-times" id="cross"></i></li>
+        </ul>
+        <ul class="mobile-menu-items mobile-menu-toggle">
+          <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+          <li><a href="#" rel="noreferrer noopener">Program</a></li>
+          <li><a href="#" rel="noreferrer noopener">Join</a></li>
+          <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+          <li><a href="#" rel="noreferrer noopener">News</a></li>
+          <li><a href="#" rel="noreferrer noopener">CC Campaign</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="home col-12">
+      <video autoplay="" muted="" loop="" id="myVideo">
+        <source
+          src="./vedio-background/Silky_Blue_720P_Motion_Background_Loop.mp4"
+          type="video/mp4"
+        />
+      </video>
+      <h4 class="welcome h4">" Welcome for Online Course "</h4>
+      <p class="det">
+        Activities and events for the new online training courses 2022
+      </p>
+      <p class="paragh">
+        The world has changed and education has become easy to obtain, so do not
+        worry, you will start a fantastic journey from which you will gain
+        knowledge and practical experience 2022
+      </p>
+      <p class="date-p">2022.01.01 (SAT) ~ 10(MON)</p>
+      <p class="address">@Haram Giza Egypt</p>
+    </section>
+    <section class="main-program col-12">
+      <h1 class="program-ma h1">
+        Main Program
+        <span class="under-l"></span>
+      </h1>
+      <div class="main-pro col-12">
+        <div class="main-box border-0">
+          <div class="border-box">
+            <i class="fas fa-laptop-code"></i>
+            <h3>Full Stack</h3>
+            <p>
+              Learn full stack web developer with html, css, javascript and
+              ruby.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fa fa-bar-chart" aria-hidden="true"></i>
+            <h3>Data Science</h3>
+            <p>
+              Learn data science and do expert for analysis and make decision.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-database"></i>
+            <h3>DataBase</h3>
+            <p>
+              Build strong database working with server and all required from
+              web or data science.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-user-graduate"></i>
+            <h3>Scholarship</h3>
+            <p>
+              Scholarship helps you to achieve your goals and many scholarship
+              here you can choose from them.
+            </p>
+          </div>
+        </div>
+        <div class="learn-more">
+          <a class="learn" href="#" rel="noreferrer noopener"
+            >More Information</a
+          >
```

Comment:
- [ ] There is a white space between the speakers and this button on the desktop. Please let's fix this.

<img width="293" alt="Loom_TGXqAmfaJC" src="https://user-images.githubusercontent.com/81468675/152480113-571ed48b-b2dd-4c83-8aec-3a445f1e414f.png">